### PR TITLE
Update last updated prefix translation

### PR DESCRIPTION
**Pull request by thekp**
Resolves: n/a

**Overall change:** 
_Our current translations for `lastUpdated` prefix, includes an extra spacing at the end, however we do not require this space as timestamp container already adds it: https://github.com/bbc/psammead/blob/latest/packages/containers/psammead-timestamp-container/src/index.jsx#L56_

**note**: https://github.com/bbc/simorgh/pull/5426/files#diff-646b7ebcab9c33a3c805840f583dffb9R24 the translation of for the Chinese services use this special colon character: `：` and not a normal colon (`:`), hence why the Chinese lastUpdate prefix looks like it has a massive sapce

**Code changes:**
- _Update translations by removing extra spacing_
- _Update unit tests/snapshots_

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by sareh — APPROVED** [review]
Looks good 👍

File: src/app/lib/config/services/igbo.js [hunk 1]
```diff
@@ -160,7 +160,7 @@ export const service = {
     },
     mostRead: {
       header: 'Akachasị Gụọ',
-      lastUpdated: 'Emelitere ikpeazụ na ',
+      lastUpdated: 'Emelitere ikpeazụ na',
```
Comment (DenisHdz) [inline src/app/lib/config/services/igbo.js]: Just wondering if this should have a `:` as well.

File: src/app/lib/config/services/pidgin.js [hunk 2]
```diff
@@ -157,7 +157,7 @@ export const service = {
     },
     mostRead: {
       header: 'De one we dem de read well well',
-      lastUpdated: 'De one we dem update for ',
+      lastUpdated: 'De one we dem update for',
```
Comment (DenisHdz) [inline src/app/lib/config/services/pidgin.js]: Same here

File: src/app/lib/config/services/somali.js [hunk 3]
```diff
@@ -164,7 +164,7 @@ export const service = {
     brandSVG,
     mostRead: {
       header: 'Ugu akhris badan',
-      lastUpdated: 'Markii ugu dambeysay ee la cusbooneysiiyay ',
+      lastUpdated: 'Markii ugu dambeysay ee la cusbooneysiiyay',
```
Comment (DenisHdz) [inline src/app/lib/config/services/somali.js]: Same here

**Review by DenisHdz — COMMENTED** [review]
A part from some services not having colon at the end, which might be expected, LGTM 👍

File: src/app/lib/config/services/hausa.js [hunk 4]
```diff
@@ -157,7 +157,7 @@ export const service = {
     brandSVG,
     mostRead: {
       header: 'Wanda aka fi karantawa',
-      lastUpdated: 'Na baya-bayan nan ',
+      lastUpdated: 'Na baya-bayan nan',
```
Comment (DenisHdz) [inline src/app/lib/config/services/hausa.js]:
```suggestion
      lastUpdated: 'Na baya-bayan nan:',
```

File: src/app/lib/config/services/yoruba.js [hunk 5]
```diff
@@ -157,7 +157,7 @@ export const service = {
     brandSVG,
     mostRead: {
       header: 'Èyítí A Ń Kà Jùlọ',
-      lastUpdated: 'Tí a mú dójú ìwọ̀n gbẹ̀yìn ní ',
+      lastUpdated: 'Tí a mú dójú ìwọ̀n gbẹ̀yìn ní',
```
Comment (DenisHdz) [inline src/app/lib/config/services/yoruba.js]:
```suggestion
      lastUpdated: 'Tí a mú dójú ìwọ̀n gbẹ̀yìn ní:',
```

**Review by DenisHdz — APPROVED** [review]

**Pull request merged**